src/parser/jobs/blm/modules/Gauge.tsx

```diff
@@ -2,36 +2,45 @@
 import {t} from '@lingui/macro'
 import {Trans, Plural} from '@lingui/react'
 import Color from 'color'
-import {ActionLink} from 'components/ui/DbLink'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
 import JOBS from 'data/JOBS'
 import {Cause, Event, Events, FieldsBase} from 'event'
-import {Analyser} from 'parser/core/Analyser'
-import {TimestampHook} from 'parser/core/Dispatcher'
 import {filter, oneOf} from 'parser/core/filter'
 import {dependency} from 'parser/core/Injectable'
 import BrokenLog from 'parser/core/modules/BrokenLog'
 import CastTime from 'parser/core/modules/CastTime'
-import {Data} from 'parser/core/modules/Data'
-import {ResourceDatum, ResourceGraphs} from 'parser/core/modules/ResourceGraphs'
+import {CounterGauge, TimerGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
 import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 import {UnableToAct} from 'parser/core/modules/UnableToAct'
-import {CompleteEvent} from 'parser/core/Parser'
 import React from 'react'
 import {isSuccessfulHit} from 'utilities'
 import {FIRE_SPELLS, ICE_SPELLS_TARGETED, ICE_SPELLS_UNTARGETED} from './Elements'
 
+/** Configuration */
 const ENOCHIAN_DURATION_REQUIRED = 30000
 export const ASTRAL_UMBRAL_DURATION = 15000
 export const MAX_ASTRAL_UMBRAL_STACKS = 3
 export const MAX_UMBRAL_HEART_STACKS = 3
 const MAX_ASTRAL_UMBRAL_CAST_SCALAR = 0.5
 const FLARE_MAX_HEART_CONSUMPTION = 3
 const MAX_POLYGLOT_STACKS = 2
```

Comment:
Can consider renaming these imo. Lead with ASTRAL_UMBRAL, ENOCHIAN, POLYGLOT, etc and use the suffix to indicate what it's used for. Mostly just nicer for intellisense, but it's easier to read.